pkgs/games/nxengine/default.nix

```diff
@@ -0,0 +1,37 @@
+{ stdenv
+, fetchurl
+, libdrm
+, SDL
+, SDL_ttf
+}:
+
+# Note that this only includes the game engine, not the game data.
+# For the game data, see the website.
+
+stdenv.mkDerivation rec {
+  name = "nxengine-${version}";
+  version = "1.0.0.6";
+  src = fetchurl {
+    url = http://nxengine.sourceforge.net/dl/nx-src-1006.tar.bz2;
+    sha256 = "0s5d445s914rr2jfmqri56ysy90mpdv4hw8af8v9kp6zvwavz76g";
+  };
+
+  buildInputs = [
+    libdrm
+    SDL
+    SDL_ttf
+  ];
+
+  installPhase = ''
+    mkdir -p $out/share/nx/
+    install -D nx $out/bin/nx
+    install -D -m 444 font.ttf sprites.sif tilekey.dat $out/share/nx/
+  '';
+
+  meta = {
+    description = "A complete open-source clone/rewrite of the masterpiece jump-and-run platformer Doukutsu Monogatari (also known as Cave Story)";
+    license = stdenv.lib.licenses.gpl3;
+    homepage = http://nxengine.sourceforge.net/;
```

Comment:
Please use quotes for the homepage if that's okay with you.

Reply:
Added quotes.
I don't have a good sense for when they're needed, since nix does seem to have URL literals.